mindsdb_native/libs/helpers/query_composer.py

```diff
@@ -0,0 +1,53 @@
+def create_history_query(query, tss, type_map, row):
```

Comment:
we need a unit test for this function

Reply:
Made it into the PR by mistake :) 
Will unit tests once the history PR is ready.